Sources/StreamChatUI/MessageActionsPopup/MessageActionsTransitionController.swift

```diff
@@ -89,7 +89,7 @@ open class ChatMessageActionsTransitionController: NSObject, UIViewControllerTra
         let messageViewFrame = selectedMessageContentViewFrame ?? .zero
         let messageViewType = type(of: originalMessageContentView)
         let messageAttachmentInjectorType = originalMessageContentView.attachmentViewInjector.map { type(of: $0) }
-        let messageLayoutOptions = originalMessageContentView.layoutOptions?.subtracting(.reactions) ?? []
+        let messageLayoutOptions = originalMessageContentView.layoutOptions?.subtracting([.reactions]) ?? []
```

Comment:
shall we introduce `subtracting` with a single item on `Set`?  🤔   if customers do that on their end it will stop compiling but the same can happen with all other set APIs

Reply:
Yes, I spotted this one... It seems to be an overkill to do that IMO, we can add all these but I think it is an Overkill and most likely it is not being used... Hopefully 🤔 Otherwise we need to add this to `subtracting` `instersection` and `insert` right?

Reply:
I've added all these backwards compatibility APIs to the Deprecations. I can also add "substracting" and "intersection" to those, WDYT?

Reply:
@evsaev Done ✅